analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -451,5 +451,301 @@ public void Finally_NestedFinally()
                 "AfterInnerFinally",
                 "AfterOuterFinally");
         }
+
+        [TestMethod]
+        public void Catch_Simple_NoFilter()
+        {
+            const string code = @"
+var tag = ""BeforeTry"";
+try
+{
+    Tag(""InTry"");
+}
+catch
+{
+    tag = ""InCatch"";
+}
+tag = ""End"";";
+            SETestContext.CreateCS(code).Validator.ValidateTagOrder(
+                "BeforeTry",
+                "InTry",
+                "InCatch",
+                "End");
```

Comment:
We need to validate the exception for the states. This will increase the visibility a lot for the exception state.

Reply:
Good catch, this caught two issues. I've added them as checkboxes and address them in another PRs.